BetterJoyForCemu/HIDapi.cs

```diff
@@ -3,11 +3,11 @@
 
 namespace BetterJoyForCemu {
 	public class HIDapi {
-		#if LINUX
+#if LINUX
 		  const string dll = "libhidapi.so";
```

Comment:
Tabs on github are like 8 spaces wide instead of 4, making misalignments like this pretty glaring. Thoughts on switching to spaces entirely?

Reply:
I'm fine with switching to spaces

Reply:
It appears it didn't format that line. supposedly because LINUX was false? i think we can disregard this problem.